src/constants.js

```diff
@@ -29,6 +29,14 @@ export const EVENT_MATCH_START = 19;
 export const EVENT_MATCH_END = 26;
 export const EVENT_VAR = 71;
 
+// Secondary Events type
+export const SHOOT = 12; // tir cadré
+export const GARDIAN_SAVE = 17; // parade
```

Comment:
D'après ce que je peux voir sur le match Suède-Suisse, ça ne concerne pas seulement le gardien.

Reply:
C'est facile de savoir si c'est le gardien grâce au champ `Position` sur les infos du joueur. Pour le gardien, c'est `0`. On peut rajouter cette information à cet endroit là https://github.com/oltodo/worldcup-for-slack/blob/master/src/Team.js#L19 :

```js
isGoalKeeper: get(player, 'Position') === 0,
```

Reply:
👍